content/en/configuration/identity-server.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Identity Server'
```

Comment:
rename all `Identity Server` to `Identity Service`

Reply:
To Identity or to Identity Service :D

Reply:
to identity store :)

Reply:
done